frontend/src/pages/articles/components/article-card/article-card.tsx

```diff
@@ -91,21 +119,40 @@ const ArticleCard: React.FC<Properties> = ({
       </div>
       <div className={styles.footer}>
         <ul className={styles.reactions}>
-          <li className={styles.reaction}>
-            <Icon iconName="comment" className={styles.pointerIcon} />
-            <span className={styles.reactionCount}>{comments}</span>
+          <li>
+            <IconButton
+              iconName="comment"
+              className={styles.footerIcon}
+              label={MOCKED_REACTIONS.comments}
+            />
           </li>
-          <li className={styles.reaction}>
+          <li className={styles.footerIcon}>
             <Icon iconName="view" />
-            <span className={styles.reactionCount}>{views}</span>
+            <span>{MOCKED_REACTIONS.views}</span>
           </li>
-          <li className={styles.reaction}>
-            <Icon iconName="like" className={styles.pointerIcon} />
-            <span className={styles.reactionCount}>{likes}</span>
+          <li>
+            <IconButton
+              iconName="like"
+              className={getValidClassNames(
+                styles.footerIcon,
+                isOwnArticle ? styles.disable : styles.reaction,
+                isLike && styles.pressed,
+              )}
+              label={String(likeCount)}
+              onClick={(): void => handleReaction(true)}
+            />
           </li>
-          <li className={styles.reaction}>
-            <Icon iconName="dislike" className={styles.pointerIcon} />
-            <span className={styles.reactionCount}>{dislikes}</span>
+          <li>
+            <IconButton
+              iconName="dislike"
+              className={getValidClassNames(
+                styles.footerIcon,
+                isOwnArticle ? styles.disable : styles.reaction,
+                isLike === false && styles.pressed,
+              )}
+              label={String(dislikeCount)}
+              onClick={(): void => handleReaction(false)}
```

Comment:
Create separate handlers that would call generic handler
eg
```
const handleDislikeReaction = () => {
  handleReaction(false);
};
```
```suggestion
              onClick={handleDislikeReaction}
```